packages/babel-plugin-ember-test-metadata/__tests__/utils-test.js

```diff
@@ -32,111 +29,120 @@ describe('getNodeProperty', () => {
 });
 
 describe('getNormalizedFilePath', () => {
```

Comment:
General comment here, but we're missing cases for _all_ the paths we need to support. We should have tests for each of the new utils created, so we've covered all test cases.

Reply:
Just re-read your comment above about adding follow ons. I'd do that but write tests for the internal utils.